activiti-core/activiti-engine/src/main/java/org/activiti/engine/delegate/event/impl/ActivitiEventBuilder.java

```diff
@@ -388,6 +389,7 @@ protected static void populateEventWithCurrentContext(ActivitiEventImpl event) {
         event.setProcessInstanceId(((Task) persistedObject).getProcessInstanceId());
         event.setExecutionId(((Task) persistedObject).getExecutionId());
         event.setProcessDefinitionId(((Task) persistedObject).getProcessDefinitionId());
+        ((ActivitiEntityEventImpl)event).setReason(TerminateEndEventActivityBehavior.createDeleteReason(null));
```

Comment:
this is hurting me

Reply:
it hurts for a reason
this should have never been committed, just a leftover from the debugging session I went through 😄